packages/types/package.json

```diff
@@ -22,7 +22,7 @@
     "@types/node": "^12.20.7",
     "@types/optimize-css-assets-webpack-plugin": "^5.0.3",
     "@types/pug": "^2.0.4",
-    "@types/sass": "^1.16.0",
+    "@types/sass-loader": "8.0.1",
```

Comment:
We can also put the package into renovate ingoreDeps for not getting auto upgradation pr

Reply:
Currently [`@types/webpack@^4` is dependency](https://unpkg.com/browse/@types/sass-loader@8.0.1/package.json) and as long as that remains we should be fine with upgrades but we'll need to examine lockfile before approving them.